src/libs/pipeline_configuration.py

```diff
@@ -0,0 +1,283 @@
+                    time_field="new_practices_reasons_time",
+                    coda_filename="new_practices_reasons.json",
+                    analysis_file_key="new_practices_reasons_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.REASONS_FOR_NEW_PRACTICES),
+
+        CodingPlan(raw_field="organizations_raw",
+                    coded_field="organizations_coded",
+                    time_field="organizations_time",
+                    coda_filename="organizations.json",
+                    analysis_file_key="organizations_reasons_",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.ORGANIZATIONS),
+    ]
+    
+    @staticmethod
+    def clean_age_with_range_filter(text):
+        """
+        Cleans age from the given `text`, setting to NC if the cleaned age is not in the range 10 <= age < 100.
+        """
+        age = swahili.DemographicCleaner.clean_age(text)
+        if type(age) == int and 10 <= age < 100:
+            return str(age)
+            # TODO: Once the cleaners are updated to not return Codes.NOT_CODED, this should be updated to still return
+            #       NC in the case where age is an int but is out of range
+        else:
+            return Codes.NOT_CODED
+  
+    DEMOGS_CODING_PLANS = [
+        CodingPlan(raw_field="age_raw",
+                    coded_field="age_coded",
+                    time_field="age_time",
+                    coda_filename="age.json",
+                    analysis_file_key="age",
+                    cleaner=lambda text: PipelineConfiguration.clean_age_with_range_filter(text),
+                    code_scheme=CodeSchemes.AGE),
+
+        CodingPlan(raw_field="constituency_raw",
+                    id_field="constituency_raw_id",
+                    coded_field="constituency_coded",
+                    time_field="constituency_time",
+                    coda_filename="constituency.json",
+                    analysis_file_key=None,
+                    cleaner=None,
+                    code_scheme=CodeSchemes.CONSTITUENCY),
+
+        CodingPlan(raw_field="gender_raw",
+                    coded_field="gender_coded",
+                    time_field="gender_time",
+                    coda_filename="gender.json",
+                    analysis_file_key="gender",
+                    cleaner=swahili.DemographicCleaner.clean_gender,
+                    code_scheme=CodeSchemes.GENDER),
+
+        CodingPlan(raw_field="livelihood_raw",
+                    coded_field="livelihood_coded",
+                    time_field="livelihood_time",
+                    coda_filename="livelihood.json",
+                    analysis_file_key="livelihood",
+                    cleaner=None,
+                    code_scheme=CodeSchemes.LIVELIHOOD),
+    ]
+    
+    def __init__(self, rapid_pro_domain, rapid_pro_token_file_url, rapid_pro_key_remappings):
+        """
+        Arguments:
```

Comment:
If you're keen to change the docstring format raise the issue on  #tech_data_team

Reply:
Will do thanks